patterns/stories-podcast-banner.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Title: Stories Podcast Hero
+ * Slug: twentytwentyfive/stories-podcast-hero
+ * Categories: featured, hero
+ * Keywords: podcast, hero, stories
+ */
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|70","bottom":"var:preset|spacing|70","left":"var:preset|spacing|70","right":"var:preset|spacing|70"}},"color":{"background":"#F6CFF4"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide has-background" style="background-color:#F6CFF4;padding-top:var(--wp--preset--spacing--70);padding-right:var(--wp--preset--spacing--70);padding-bottom:var(--wp--preset--spacing--70);padding-left:var(--wp--preset--spacing--70)"><!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|60","left":"var:preset|spacing|60"}}}} -->
+	<div class="wp-block-columns alignwide"><!-- wp:column {"width":"40%","layout":{"type":"default"}} -->
+		<div class="wp-block-column" style="flex-basis:40%"><!-- wp:image {"className":"wp-block-image size-large is-resized"} -->
+			<figure class="wp-block-image size-large is-resized"><img src="<?php echo esc_url(get_template_directory_uri()); ?>/assets/images/image-from-rawpixel-id-14264288-jpeg.png" alt="Historical figure" /></figure>
```

Comment:
Let's fix the spacing of the PHP functions. If possible, let's also try to use a `.webp` image.

Also, the aspect ratio should be square for the image.

Reply:
hey @juanfra 
circling back to this

Actioned all the feedback but I diont have an actual version of this image I ripped it from the figma file with a screengrabber. any chance there is a repo somewhere of images for this? reverse google lookups are all rectangular - is it ok in this regard to just use another screengrab thats just... better? lol